services/ui-src/src/utils/validation/schemas.test.ts

```diff
@@ -38,7 +37,7 @@ describe("Schemas", () => {
   ];
 
   const testNumberSchema = (
-    schemaToUse: MixedSchema<any, AnyObject, any>,
+    schemaToUse: MixedSchema,
```

Comment:
Question: I'm curious why we made this change since it's the only change to our frontend validation schema. Was it meant to be made to our backend validation schema? If so, do we need to make the same change to our frontend schema?

Reply:
yeah this is fine.